src/scripts/main.js

```diff
@@ -1 +1,80 @@
 'use strict';
+
+document.addEventListener('contextmenu', (e) => e.preventDefault());
```

Comment:
The event listener for 'contextmenu' prevents the default context menu from appearing. Ensure this is intended, as it affects all right-clicks on the page.